README.md

```
@@ -130,27 +145,89 @@ TalentMesh **solves these problems** by offering an **on-demand interview outsou
 <img src="public/images/ER-Diagram.png" alt="ER Diagram" width="100%">
 
 
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)